package.json

```diff
@@ -10,7 +10,8 @@
     "test:browser": "standard && npm run deps && NODE_ENV=test zuul tests/browser/*",
     "test:browser:local": "standard && npm run deps && NODE_ENV=test zuul --local 8080 -- tests/browser/*",
     "test:cov": "npm run test:server:cov && npm run test:browser",
```

Comment:
haha, was wondering why tests were still failing - might need to update this line too, as this is what runs through Travis 😋